DepositApi/Controllers/GetCSVDepositApiController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using DepositApi.BLL.Intrerfaces;
+using DepositApi.DAL.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DepositApi.Controllers
+{
+    [ApiController]
+    [Route("/api/depositcalculation/csv")]
+    public class GetCSVDepositApiController : ControllerBase
```

Comment:
Why CSV?

Reply:
Because as a result of this request we get a CSV file